src/main/java/Task.java

```diff
@@ -0,0 +1,33 @@
+public class Task {
+    protected String task;
+    protected boolean isDone;
+
+    public Task (String task) {
+        this.task = task;
+        this.isDone = false;
+    }
+
+    public void markDone() {
+        this.isDone = !isDone;
+    }
+
+    public String getTaskType() {
+        return "";
+    }
+
+    public String getDeadline() {
+        return "";
+    }
+
+    public String getStart() {
+        return "";
+    }
+
+    public String getEnd() {
+        return "";
+    }
```

Comment:
you can consider making Task.java an abstract class. https://nus-cs2113-ay2324s2.github.io/website/schedule/week5/topics.html#w5-2-oop-java-abstract-classes